packages/components/src/internal/components/domainproperties/issues/IssuesListDefPropertiesPanelFormElements.tsx

```diff
@@ -112,6 +125,40 @@ export class AssignmentOptions extends PureComponent<AssignmentOptionsProps, Ass
     }
 }
 
+export class RestrictedOptions extends PureComponent<RestrictedOptionsProps, RestrictedOptionsState> {
+    state: Readonly<RestrictedOptionsState> = {coreGroups: undefined,};
+
+    componentDidMount = async (): Promise<void> => {
+        try {
+            const coreGroups = await getProjectGroups();
+            this.setState({ coreGroups });
+        } catch (e) {
+            console.error('RestrictedOptions: failed to load initialize project groups', e);
+        }
+    };
+
+    render() {
+        const { model, onCheckChange, onSelect } = this.props;
+        const { coreGroups } = this.state;
+
+        return (
+            <div>
+                <SectionHeading title="Restricted List Options" />
+                <RestrictedIssueInput
+                    model={model}
+                    onCheckChange={onCheckChange}
+                    onSelect={onSelect}
```

Comment:
this `RestrictedIssueInput` is just the checkbox right? I assume it doesn't actually need or use the `onSelect`. Can that be removed here?

Reply:
I found I needed that because both the input and select extend `RestrictedOptionsProps` which specifies that property. Is there a more efficient way to handle this?